test/features/test_opensmile.py

```diff
@@ -0,0 +1,108 @@
+from tempfile import NamedTemporaryFile
+import numpy as np
+import pytest
+from lhotse import Recording, OpenSmileWrapper, OpenSmileConfig
+from lhotse.utils import is_module_available, compute_num_frames, compute_num_samples
+

```

Comment:
you can add the following line and remove all skipif marks and local imports:

```python
opensmile = pytest.importorskip("opensmile", reason=...)
```

Reply:
Agree. Done.